components/brave_sync/sync_devices.h

```diff
@@ -43,6 +43,7 @@ class SyncDevices {
    std::unique_ptr<base::Value> ToValueArrOnly() const;
    std::string ToJson() const;
    size_t size() const { return devices_.size(); }
+   bool have_less_than_two() const { return size() < 2; }
```

Comment:
just a nit, but can we make this `has_second_device` and reverse the logic? I think the purpose would be more intuitive and `have_less_than_two` just sounds weird to me